src/schema/tags.ts

```diff
@@ -76,5 +112,46 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
         hits: hits.map((x) => ({ name: x.value })),
       };
     },
+    onboardingTags: async (source, args, ctx): Promise<GQLTagResults> => {
+      const hits = await ctx.getRepository(Keyword).find({
+        select: ['value'],
+        where: {
+          flags: {
+            onboarding: true,
+          },
+        },
+        order: { value: 'ASC' },
+      });
+
+      return {
+        hits: hits.map((hit) => ({ name: hit.value })),
+      };
+    },
+    recommendedTags: async (
+      source,
+      { tags, excludedTags },
+      ctx,
+    ): Promise<GQLTagResults> => {
+      const uniqueTagsToExclude = new Set([...tags, ...excludedTags]);
+
+      if (uniqueTagsToExclude.size > 1000) {
+        throw new ValidationError('Tag limit reached');
+      }
```

Comment:
Added this as a guard because potentially someone could send really big payloads which would result in big queries to the DB.